src/Admin/DonationAdmin.php

```diff
@@ -86,6 +76,20 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
             ->add('lastName', null, [
                 'label' => 'Nom',
             ])
+            ->add('emailAddress', null, [
+                'label' => 'Adresse e-mail',
+            ])
+            ->add('status', 'doctrine_orm_choice', [
+                'label' => 'Status',
+            ], ChoiceType::class, [
```

Comment:
CD: need an indent.

Otherwise you ca simplify like this :

```php
->add(
    'status', 
    'doctrine_orm_choice',
    ['label' => 'Status'],
    ChoiceType::class,
    [
        'choices' => [
            'En attente' => Donation::STATUS_WAITING_CONFIRMATION,
            'Souscription en cours' => Donation::STATUS_SUBSCRIPTION_IN_PROGRESS,
            'Erreur' => Donation::STATUS_ERROR,
            'Annulé' => Donation::STATUS_CANCELED,
            'Fini' => Donation::STATUS_FINISHED,
        ],
    ]
)
```